### PR TITLE
Some fixes and changes to IFV

### PR DESCRIPTION
**Pull request by MustaphaTR**
Depends on #420

### CODE REVIEW

**Comment by Mailaender** [comment]
Needs a rebase.

**Comment by MustaphaTR** [comment]
Rebased.

**Comment by Mailaender** [comment]
Can you write less generic commit messages?

**Comment by MustaphaTR** [comment]
I'll add Yuri IFV to this too, as Yuri PR is now merged. But i completed seperating the commits. Hope haven't messed something up.

**Comment by MustaphaTR** [comment]
Updated.

File: mods/ra2/weapons/misc.yaml [hunk 1]
```diff
@@ -69,3 +69,11 @@ PsiWave:
 		DamageTypes: PsychicDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: psi_wave
+
+CRPsiWave:
+	Inherits: PsiWave
+	ReloadDelay: 20
+	Projectile: Missile
+		RangeLimit: 0c1
```
Comment (Mailaender) [inline mods/ra2/weapons/misc.yaml]: > Error: Weapon `crpsiwave`: projectile RangeLimit lower than weapon range!

Comment (GraionDilach) [reply mods/ra2/weapons/misc.yaml]: Since when this produces an error?! That's stupid.

Comment (GraionDilach) [reply mods/ra2/weapons/misc.yaml]: Ahwell, that means OpenRA needs my InstantExplode projectile longterm. How unsurprising.

Comment (MustaphaTR) [reply mods/ra2/weapons/misc.yaml]: Whatever, i'll drop the Yuri IFV, this is blocking Auto Target PR and it is more important.

**Comment by MustaphaTR** [comment]
I have removed the Yuri IFV for now. This should be good to go.

**Pull request merged**